homeassistant/components/recorder/migration.py

```diff
@@ -457,6 +464,13 @@ def _apply_update(engine, session, new_version, old_version):
             # Recreate the statistics table
             Statistics.__table__.drop(engine)
             Statistics.__table__.create(engine)
```

Comment:
We can just remove the #17 schema changes from here (and replace it with a comment why it is empty).

Simply because #18 will do the same again :)

Reply:
Thanks!🤦